src/components/executionExpirationtime/index.tsx

```diff
@@ -0,0 +1,269 @@
+                value={value}
+                onBlur={onBlur}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                  handleMinutesChanged(e, onChange)
+                }
+                placeholder={'0'}
+                min="0"
+                disabled={
+                  executionExpirationDays ===
+                  COMMITTEE_EXECUTION_MAX_DURATION_DAYS.toString()
+                }
+              />
+              {error?.message && (
+                <AlertInline label={error.message} mode="critical" />
+              )}
+            </TimeLabelWrapper>
+          )}
+        />
+
+        <Controller
+          name="executionExpirationHours"
+          control={control}
+          defaultValue="0"
+          rules={{required: t('errors.emptyDistributionHours')}}
+          render={({
+            field: {onBlur, onChange, value, name},
+            fieldState: {error},
+          }) => (
+            <TimeLabelWrapper>
+              <TimeLabel>{t('createDAO.step4.hours')}</TimeLabel>
+              <NumberInput
+                name={name}
+                value={value}
+                onBlur={onBlur}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                  handleHoursChanged(e, onChange)
+                }
+                placeholder={'0'}
+                min="0"
+                disabled={
+                  executionExpirationDays ===
+                  COMMITTEE_EXECUTION_MAX_DURATION_DAYS.toString()
+                }
+              />
+              {error?.message && (
+                <AlertInline label={error.message} mode="critical" />
+              )}
+            </TimeLabelWrapper>
+          )}
+        />
+
+        <Controller
+          name="executionExpirationDays"
+          control={control}
+          defaultValue="1"
+          rules={{
+            required: t('errors.emptyDistributionDays'),
+            validate: value =>
+              value >= 0 ? true : t('errors.distributionDays'),
+          }}
+          render={({
+            field: {onBlur, onChange, value, name},
+            fieldState: {error},
+          }) => (
+            <TimeLabelWrapper>
+              <TimeLabel>{t('createDAO.step4.days')}</TimeLabel>
+              <NumberInput
+                name={name}
+                value={value}
+                onBlur={onBlur}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                  handleDaysChanged(e, onChange)
+                }
+                placeholder={'0'}
+                min="0"
+              />
+              {error?.message && (
+                <AlertInline label={error.message} mode="critical" />
+              )}
+            </TimeLabelWrapper>
+          )}
+        />
+      </DurationContainer>
+      {executionExpirationDays ===
+      COMMITTEE_EXECUTION_MAX_DURATION_DAYS.toString() ? (
+        <AlertInline
+          label={t('alert.maxDurationAlert') as string}
```

Comment:
The cast to string is redundant here and below as well, does it throws some type error for you?
```suggestion
          label={t('alert.maxDurationAlert')}
```